### DISCUSSION TITLE
Engine trails

### DISCUSSION THREAD

**Discussion by cloudrambler** [tsunamayo/Starship-EVO#4078] [Suggestions]
Can we have engine trails and thruster plumes?

Engine trails would be lines traced out by lightcruise engines that trace the path followed by the ship. They should fade after some distance.

Thruster plumes would be the local jets emitted by lateral thrusters. Ideally these would correspond to logical manoeuvring forces. I recall this was a cool feature of Elite2. I know the physics model is simplified, because correct thruster placement is not required to gain effectiveness, but the proportional thrust vectoring model could still be applied for purely aesthetic purposes. Then for those of us who place thrusters carefully, the effect would look realistic.

I suggest both effects have a toggle to disable them, as some may prefer the "reactionless" aesthetic.

Side question: will there ever be a bonus for forward facing thrusters? Theoretically you cannot slow down without them, so I wonder could their inclusion provide a tangible benefit, such as improved deceleration.

**Comment by ExodistSKY1** [comment]
I would like to have larger thruster plumes as a visual aspect to the game!

**Comment by Jorpis-Jorah** [comment]
When the game was still skywanderers, it had something similar, I would like to see them again

**Comment by tsunamayo** [comment]
Yes I had these in the past, but it was too buggy and expensive. I would have to roll my own solution instead. As for benefit for correct placement this is not the way I wanted to go gameplay wise. thx

**Comment by ExodistSKY1** [comment]
These are a few images of what I have in mind when thinking of engine trails

![GetAttachmentThail](https://user-images.githubusercontent.com/45324484/130309274-1338550c-596a-46f1-85a9-c0ca872cdd19.png)
![GetAttachmentil](https://user-images.githubusercontent.com/45324484/130309276-79ad03e3-0310-46f8-92ae-399423a4812e.png)
![GetAttachmentThumbnail](https://user-images.githubusercontent.com/45324484/130309280-9e89e640-651f-4463-a5eb-d22fb20fae2c.png)

They do not need to be "Homeworld" style but having larger plumes from the engines would be a very nice visual and help with identifying ships that are in motion
